pkg/detectors/autoklose/autoklose.go

```diff
@@ -50,23 +51,38 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", fmt.Sprintf("https://api.autoklose.com/api/campaigns/?api_token=%s", resMatch), nil)
+			// API Documentation: https://api.aklab.xyz/#auth-info-fd71acd1-2e41-4991-8789-3edfd258479a
+			req, err := http.NewRequestWithContext(ctx, "GET", fmt.Sprintf("https://api.autoklose.com/api/me/?api_token=%s", resMatch), nil)
 			if err != nil {
 				continue
 			}
+			req.Header.Add("Accept", "application/json")
 			res, err := client.Do(req)
 			if err == nil {
+				defer res.Body.Close()
 				bodyBytes, err := io.ReadAll(res.Body)
 				if err != nil {
 					continue
 				}
-				defer res.Body.Close()
-				if res.StatusCode >= 200 && res.StatusCode < 300 {
-					if json.Valid(bodyBytes) {
-						s1.Verified = true
-					} else {
-						s1.Verified = false
+				if res.StatusCode == 401 {
+					s1.Verified = false
+				}
+
+				if res.StatusCode == 200 {
+					s1.Verified = true
+				}
+
+				if json.Valid(bodyBytes) {
+					var responseBody map[string]interface{}
+					if err := json.Unmarshal(bodyBytes, &responseBody); err == nil {
+						if email, ok := responseBody["email"].(string); ok {
+							s1.ExtraData = map[string]string{
+								"email": email,
+							}
+						}
 					}
+				} else {
+					s1.Verified = false
```

Comment:
Verified by default is unverified so not required to mark it explicitly. Following code should work fine. You can cross check one time. 
```
res, err := client.Do(req)
if err == nil {
    defer func() {
        _, _ = io.Copy(io.Discard, res.Body)
	_ = res.Body.Close()
    }()

if res.StatusCode == http.StatusOK {
    s1.Verified = true
    bodyBytes, err := io.ReadAll(res.Body)
    if err != nil {
        continue
    }

var responseBody map[string]interface{}
if err := json.Unmarshal(bodyBytes, &responseBody); err == nil {
    if email, ok := responseBody["email"].(string); ok {
        s1.ExtraData = map[string]string{
	    "email": email,
	    }
	}
    }
}
} else {
    s1.SetVerificationError(err, resMatch)
}
```

Reply:
Thanks made the change